UI/cypress/e2e/results.feature

```diff
@@ -12,6 +12,21 @@ Feature: User is shown search results
     When they click the 'Find a tuition partner' service name link
     Then they will be taken to the 'Find a tuition partner' journey start page
 
+  Scenario: Back click returns to subjects page
+    Given a user has arrived on the 'Search results' page for 'Key stage 1 English' for postcode 'SK1 1EB'
+    When they click 'Back'
+    Then they will be taken to the 'Which subjects' page
+    And they are shown the subjects for 'Key stage 1'
+    And they will see 'Key stage 1 English' selected
+
```

Comment:
Worth adding the back to which key stages test as you've done the back to postcode one?